### PR TITLE
fixed descriptions for galaxy, tacc, ucsc, and ncbi

### PR DESCRIPTION
**Pull request by nekrut**

### CODE REVIEW

File: app/components/Home/components/Section/components/SectionAnalytics/components/AnalyticsTools/common/constants.ts [hunk 1]
```diff
@@ -5,8 +5,12 @@ const CARD_ACTION_LABEL = {
   LEARN_MORE: "Learn More",
 };
 const CARD_TEXT = {
-  RUN_BATCH_ANALYSIS_WORKFLOWS:
-    "Run batch analysis workflows and interactive visualizations.",
+  ABOUT_GALAXY:
+    "A platform for the analysis of data at any scale using public infrastructure",
+  ABOUT_NCBI: "A singular source for standardized genomic and functional data.",
```
Comment (dannon) [inline app/components/Home/components/Section/components/SectionAnalytics/components/AnalyticsTools/common/constants.ts:10]:
```suggestion
  ABOUT_NCBI: "A singular source for standardized genomic and functional data",
```

**Comment by dannon** [comment]
(pushed minor tweak standardizing short description punctuation)

File: app/components/Home/components/Section/components/SectionAnalytics/components/AnalyticsTools/common/constants.ts [hunk 2]
```diff
@@ -5,8 +5,12 @@ const CARD_ACTION_LABEL = {
   LEARN_MORE: "Learn More",
 };
 const CARD_TEXT = {
-  RUN_BATCH_ANALYSIS_WORKFLOWS:
-    "Run batch analysis workflows and interactive visualizations.",
+  ABOUT_GALAXY:
+    "A platform for the analysis of data at any scale using public infrastructure",
+  ABOUT_NCBI: "A singular source for standardized genomic and functional data",
+  ABOUT_TACC: "A world's leading academic computing center",
```
Comment (dannon) [inline app/components/Home/components/Section/components/SectionAnalytics/components/AnalyticsTools/common/constants.ts:11]:
```suggestion
  ABOUT_TACC: "One of the world’s leading academic computing centers",
```

**Comment by nekrut** [comment]
@dannon -> that you for checking, "me no speak english" ;)

**Review by NoopDog — APPROVED** [review]

**Comment by NoopDog** [comment]
@nekrut this is deployed to prod. Thx!

**Pull request merged**